keyboards/keebio/nyquist/keymaps/default_4x12/keymap.c

```diff
@@ -0,0 +1,140 @@
+#include QMK_KEYBOARD_H
+
+enum layer_names {
+    _QWERTY,
+    _LOWER,
+    _RAISE,
+    _FUNCTION,
+    _ADJUST,
+};
+
+enum planck_keycodes {
+  QWERTY = SAFE_RANGE,
```

Comment:
The `QWERTY` keycode is in the keymap, but doesn't have any logic written for it. It should probably be removed from the keymap and the definition deleted.

```suggestion
```